ios/NotificationsHelperIOS10.swift

```diff
@@ -59,11 +60,14 @@ class NotificationsHelperIOS10: NSObject, UNUserNotificationCenterDelegate {
     }))
   }
   
-  private func showRichNotification(title: String, subtitle: String, body: String, url: String) {
+  private func showRichNotification(title: String, subtitle: String, body: String, image: String?, url: String) {
     let content = UNMutableNotificationContent()
     content.title = title
     content.subtitle = subtitle
     content.body = body
+    if let image: String = image! {
```

Comment:
I could use a different variable name like _image, but that pattern is quite used in swift to be sure that you get the value that you want from an Optional.